File: twilio/sample-code-master/api/v2010/incoming_phone_number/read-example-2/read-example-2.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$incomingPhoneNumbers = $twilio->incomingPhoneNumbers
                               ->read(array("phoneNumber" => "+14158675310"));

foreach ($incomingPhoneNumbers as $record) {
    print($record->sid);
}

File: twilio/sample-code-master/api/v2010/conference/read-example-2/read-example-2.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

conferences = @client.conferences.list(
                                    friendly_name: 'MyRoom',
                                    status: 'in-progress'
                                  )

conferences.each do |record|
  puts record.sid
end
